backend/rest_api_service/crud.py

```diff
@@ -10,6 +11,61 @@ async def get_all_users(db: Session) -> list[Type[DBUser]] | None:
     return users
 
 
+
+ 
+async def get_user_by_id(db: Session, user_id: int) -> models.DBUser | None:
+    user = db.query(models.DBUser).filter(models.DBUser.user_id == user_id).first()
+    print(user)
+    
+    return user
+
+async def get_user_by_username(db: Session, username: str) -> models.DBUser | None:
+    user = db.query(models.DBUser).filter(models.DBUser.username == username).first()
+    print(user)
+    
+    return user
+
+async def get_all_events_by_user_id(db: Session, user_id: int) -> models.DBUser | None:
```

Comment:
Czemu robimy tutaj `db.query(models.DBUser)` nie możemy zrobić `db.query(models.DBEvent)` i potem przefiltrować po user id?
Jakoś zbyt skomplikowanie tutaj to wygląda xd
Plus tutaj trzeba dodać try/catch